File: templates/Uwp/_comp/CodeBehind/Project/Activation/DefaultLaunchActivationHandler_postaction.cs
namespace Param_RootNamespace.Activation
{
    internal class DefaultLaunchActivationHandler : ActivationHandler<LaunchActivatedEventArgs>
    {
//{[{
        private readonly Type _navElement;

        public DefaultLaunchActivationHandler(Type navElement)
        {
            _navElement = navElement;
        }

//}]}
    }
}

File: templates/Uwp/_comp/CodeBehind/Project._VB/Activation/DefaultLaunchActivationHandler_postaction.vb
Namespace Activation
    Friend Class DefaultLaunchActivationHandler
        Inherits ActivationHandler(Of LaunchActivatedEventArgs)
'{[{

        Private ReadOnly _navElement As Type

        Public Sub New(navElement As Type)
            _navElement = navElement
        End Sub
'}]}
    End Class
End Namespace
